README.md

```diff
@@ -39,6 +39,14 @@ The **Tooling for Apache Camel K** extension is available in the VS Code Extensi
 
     You can install them by following the same steps except search for Kubernetes or Apache Camel in the list of extensions.
 
+## get started using Didact tutorial
+
+A tutorial to get started is available when [VS Code Didact](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-didact) is installed. It is accessible through the `Didact Tutorials` Tree view:
```

Comment:
I am not a friend of calling it Tree view. It is on first sight just a view. Tree is an implementation detail users don't really care about.